style.css

```diff
@@ -78,7 +89,7 @@
   font-size: 16px;
```

Comment:
- [ ] According to the figma design the font-size for this intro paragraph should be 20px :point_down: 
![image](https://user-images.githubusercontent.com/97521557/230734429-2b8b06ab-9c4e-46b5-82b3-00ebd3d4159c.png)